docs/node-operators/get-started/setup-datanode.md

```diff
@@ -641,11 +683,59 @@ The GraphQL will return error for queries that have complexity above the set lim
 ### REST
 REST provides a standard between computer systems on the web, making it easier for systems to communicate with each other. It is arguably simpler to work with than gRPC and GraphQL. In Vega the REST API is a reverse proxy to the gRPC API, however it does not support streaming.
 
-The default port (configurable) for the REST API is `3009` and we use a reverse proxy to the gRPC API to deliver the REST API implementation.
+The default port (configurable) for the REST API is `3008` and we use a reverse proxy to the gRPC API to deliver the REST API implementation. You can change the port with the `Gateway.Port` parameter in your `config.toml`.
 
 ## Further reading
 For more information about data node and developing on data node please see the data node [README ↗](https://github.com/vegaprotocol/vega/blob/master/datanode/README.md)
 
+### Testing your data-node public setup
+
+It is required to expose the following things from the data node:
+
+- GraphQL API
+- REST API
+- GRPC API
+
+Rest endpoints can be closed.
+
+You must expose the GraphQL and the REST API on your domain with a valid TLS certificate. It means you either 
+
+- set the 443 port for the data node and set up the auto cert for your data node(see instruction above)
+- set the reverse proxy for the `Gateway.Port` in external software like Nginx/Apache/Caddy.
+
+You also have to open the GRPC port on your server; by default, it is `3007`. You can change the GRPC port with the `API.Port` parameter in your data node `config.toml`.
+
+Assuming you have correct setup for the Gateway port(REST & GQL APIs), and the API port set to 3007, and domain `api` points to your server, both requests should return the correct response:
```

Comment:
Shouldn't this port be 3008 as it's for GraphQL and REST?

Reply:
We want validators to expose those services on port 443(which is HTTPS). They can do it at least in 2 ways:
- Use 443 directly in data-node 
- Proxy it thru some web server like nginx/caddy

Reply:
Ahh i see it is confusing. I will update it